explorer/syntax/lexer.lpp

```diff
@@ -101,7 +101,7 @@ WHILE                "while"
 /* This should be kept table-like, but isn't automatic due to spaces. */
 identifier            [A-Za-z_][A-Za-z0-9_]*
 /* TODO: Remove Print special casing once we have variadics or overloads. */
-intrinsic_identifier  (Print|__intrinsic_[A-Za-z0-9_]*)
+intrinsic_identifier  (Print|Rand|__intrinsic_[A-Za-z0-9_]*)
```

Comment:
I'm afraid the way we're handling intrinsics is in a really misleading state right now. Intrinsics should have names starting with `__intrinsic_`, and shouldn't be called directly in user code. Instead, user code should call a function that's defined in `prelude.carbon`, which calls the intrinsic. That way, for example, we don't need to write a bunch of special-case typechecking logic for the intrinsic, because we can manually make sure the `prelude.carbon` function is correct, and then the ordinary typechecking logic will handle calls to the function.

Unfortunately, we don't currently have any examples of that. In #1407 we decided to break those rules for `Print` so that we could make it variadic and overloaded in time for the public announcement. However, you take a look at that PR to get a sense of how it used to work. I think the way `Print` used to work is the way that `Rand` should work now. Does that make sense?

Reply:
Yes it make sense and you are reight! I have implemented it!
I'm a novice in carbon implementation and this kind of comments are very usefull for me, thanks!